app/assets/javascripts/welcome.coffee

```diff
@@ -35,3 +35,17 @@ $ ->
       success: (data) ->
         $('#partial_news').html data
     event.preventDefault()
+
+  $('#myCarousel').carousel()
+  item = $('.carousel .item')
+  item.eq(0).addClass('active')
+  numberofSlides = $('.item').length
+  currentSlide = Math.floor(Math.random() * numberofSlides)
```

Comment:
'currentSlide' is just variable name I can rename it to "slide". Carusel starts from first photo  item.eq(0).addClass('active')